### ISSUE TITLE
As a person who perfers ease of use, I want the website to be intuitive to navigate so I can use the website with relative ease 

### ISSUE THREAD

**Issue by stevenAdrianCarter** [xlab-classes/cse442-spring2022-team-seashell-anatomy#17]
**Acceptance Tests**

*Test 1*
1. View the homepage of the website
2. Verify that the homepage has the following links:
  a. Homepage
  b. Playlist Generator
  c. About

*Test 2*
1. Navigate to the about page
2. Ensure that there are 5 sections, one for each member of the team
3. Ensure that each section has the following sections:
  a. Image
  b. Name
  c. Major

*Test 3*
1. Navigate to the Playlist Generator page
2. Ensure that a form is in the center of the page
3. Ensure that each of the following parameters exists:
    a. acousticness
    b. danceability
    c. energy
    d. valence
 4. Ensure that all categories are formatted as a "range" input (a slider)

**Comment by aleena-sheikh** [comment]
task test is too vague and not quantifiable, how would it fail?

**Comment by aleena-sheikh** [comment]
remove assignee, blocked by new task , move to in progress

**Comment by aleena-sheikh** [comment]
figma should have a results page, see possible drop down, add submit button, add rating form in output
